Gemfile

```diff
@@ -16,7 +16,7 @@ source 'https://rubygems.org'
 
 group :test do
   gem 'rake'
-  gem 'puppet', ENV['PUPPET_VERSION'] || '~> 3.8.0'
+  gem 'puppet', ENV['PUPPET_VERSION'] || '~> 4.0'
```

Comment:
The Gemfile is (or will be) managed by modulesync.

https://github.com/voxpupuli/modulesync_config

Please raise a PR over there about it, or this will get overwritten the next time we sync.